tests/unit_tests/dft/include/test_common.hpp

```diff
@@ -162,6 +162,17 @@ void commit_descriptor(oneapi::mkl::dft::descriptor<precision, domain> &descript
 #endif
 }
 
+template <typename Func, typename... Args>
+void dispatch(sycl::queue queue, Func func, Args &&... args) {
+#ifdef CALL_RT_API
+    (void)queue;
+    func(std::forward<Args>(args)...);
+#else
+    // can't get return from this, so func must capture return values by reference in a lambda
+    TEST_RUN_CT_SELECT(queue, func, std::forward<Args>(args)...);
```

Comment:
Is this working for the dft domain? We've had to introduce `TEST_RUN_CT_SELECT_NO_ARGS` to use cufft for instance.

Reply:
I added the `TEST_RUN_CT_SELECT` previously, so it works fine https://github.com/oneapi-src/oneMKL/blob/055b3701c0b1eeb7a3c33866cd9239f0ddb1015c/tests/unit_tests/include/test_helper.hpp#L175
I can post a cufft log as well

Reply:
Thanks, my bad I missed it.